### DISCUSSION TITLE
Remove "Promote" UI

### DISCUSSION THREAD

**Discussion by bnb** [typefully/minimal-twitter#93] [Ideas] [answered]
if you turn your account into a "Professional" account, Twitter aggressively tries to get you to promote your tweets as you put them out. while I'm sure they have their reasons for doing this, I'm literally never going to promote a tweet and it'd be sick if I could just hide that feature.

![image](https://user-images.githubusercontent.com/502396/155816996-15b912b2-a99c-4d50-9979-cbb79c30071c.png)

**Comment by ghost** [comment]
You can use uBlock Origin's picker mode to remove the elemet

**Comment by ghost** [comment, marked as answer]
Thanks @bnb added to `4.2.0`: https://github.com/thomaswang/minimal-twitter/releases/tag/v4.2.0
